src/components/PopupWithForm.js

```diff
@@ -8,6 +11,16 @@ export class PopupWithForm extends Popup {
   close() {
     super.close();
     this.form.reset();
+    //
+
+    const errorMessages = this.form.querySelectorAll(".form__input-error");
+    errorMessages.forEach((errorMessage) => {
+      errorMessage.classList.remove("form__input-error_active");
+    });
+    const inputs = this.form.querySelectorAll(".form__input");
+    inputs.forEach((input) => {
+      input.classList.remove("form__input_type_error");
+    });
   }
   _getInputValues() {
     const inputList = this.form.querySelectorAll(".form__input");
```

Comment:
**Could be improved**

All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `getView`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method or search them again because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` , `this._likeButton` and so on.